README.md

```diff
@@ -210,10 +212,14 @@ Now the fun part - writing to the log!
 The `AppendOptions` allow Tessera behaviour to be tuned.
 Take a look at the methods named `With*` on the `AppendOptions` struct in the root package, e.g. [`WithBatching`](https://pkg.go.dev/github.com/transparency-dev/trillian-tessera@main#AppendOptions.WithBatching) to see the available options are how they should be used.
 
-Whichever storage option you use, writing to the log follows the same pattern: simply call `Add` with a new entry created with the data to be added as a leaf in the log.
-This method returns a _future_ of the form `func() (idx uint64, err error)`.
-When called, this future function will block until the data passed into `Add` has been sequenced and an index number is _durably_ assigned (or until failure, in which case an error is returned).
-Once this index has been returned, the new data is sequenced, but not necessarily integrated into the log.
+Writing to the log follows this flow:
+ 1. Call `Add` with a new entry created with the data to be added as a leaf in the log.
+    - This method returns a _future_ of the form `func() (idx uint64, err error)`.
```

Comment:
Actually, this needs updating - it returns a `func()  (Index, err)` now, I think.

Reply:
Fixed